core/src/main/scala/scalacss/Css.scala

```diff
@@ -53,14 +58,16 @@ object Css {
   type ValuesByMediaQuery = NonEmptyVector[(CssSelector, NonEmptyVector[CssKV])]
   type ByMediaQuery       = Map[CssMediaQueryO, ValuesByMediaQuery]
 
-  def findStylesAndAnimations(c: Css): (Stream[CssStyleEntry], Stream[CssKeyframesAnimation]) = {
+  def findStylesAndAnimationsAndFontFaces(c: Css): (Stream[CssStyleEntry], Stream[CssKeyframesAnimation], Stream[CssFontFace]) = {
```

Comment:
can that be decoupled into (if that makes sense):
- findSyles
- findAnimations
- findFontFaces
- findAllTheStuff
?